lib/player.rb

```diff
@@ -0,0 +1,12 @@
+# Player has players information
+# Player has name, symbol, winner and moves methods
+class Player
+  attr_accessor :name, :symbol, :winner, :moves
```

Comment:
Which of these attributes do you need to change in the course of the game? Only those attributes that will change (i.e, you will be reassigning) should be in `attr_accessor`